Cambridge Core.js

```diff
@@ -156,6 +156,10 @@ function scrape(doc, url) {
 			if (item.date.includes("undefined")) {
 				item.date = attr('meta[name="citation_online_date"]', "content");
 			}
+			// remove asterisk or 1 at end of title, e.g. https://www.cambridge.org/core/journals/american-political-science-review/article/abs/violence-in-premodern-societies-rural-colombia/A14B0BB4130A2BA6BE79E2853597526E
```

Comment:
wouldn't it be better to actually test for the string at the end of the title? I see two potential issues here:
1. If Cambridge ever fixes their metadata and remove the asterisk from `citation_title`, we'll randomly chop of the last character
2. This won't work properly if there are multiple hyperlinked characters at the end of a title (which I don't have an example for, but is generally not implausible in scholarly publishing.

Alternatively, we could consider getting/replacing the title straight from h1 and just remove the hyperlinked content.